src/index.js

```diff
@@ -51,9 +51,7 @@ class Redactyl {
 
       if (Array.isArray(redacted[prop])) {
         redacted[prop].forEach((value, index) => {
-          if (this.isObject(value)) {
-            redacted[prop][index] = this.redact(value);
-          }
+          redacted[prop][index] = this.redact(value);
```

Comment:
Just added code coverage into the project in one my open PRs and noticed this line wasn't being covered. Thanks for the fix and good catch! :innocent: